### PR TITLE
Add created, created_by columns to artifacts

### PR DESCRIPTION
**Pull request by melissachang**
Add created, createdBy columns to Cohort, Concept Set, Cohort Review, Study.

Also added last_modified to artifacts that were missing it.

I noticed when I update cohort (change name), last_modified isn't updated. That's because [this method](https://github.com/DataBiosphere/tanagra/blob/952892907fdf67f8929a111a65fa94141a5dea8d/service/src/main/java/bio/terra/tanagra/db/CohortDao.java#L307) doesn't set last_modified. I'll follow up with Mariko after the holidays.

Manually tested:
- New fields returned

image

- created/created_by don't change after update
- For created_by: Existing rows (rows that existed before changeset) have "unknown". New rows are set to user email.

### CODE REVIEW

File: package-lock.json
Comment (marikomedlock) [inline package-lock.json:1]: Is this an `npm` generated file? I would have expected it to put things in the `ui` directory, not at the top-level.

File: service/src/main/java/bio/terra/tanagra/db/ReviewDao.java
Comment (marikomedlock) [inline service/src/main/java/bio/terra/tanagra/db/ReviewDao.java:94]: `.addValue("createdBy", review.getCreatedBy());`?

File: service/src/main/java/bio/terra/tanagra/db/ReviewDao.java
Comment (marikomedlock) [inline service/src/main/java/bio/terra/tanagra/db/ReviewDao.java:51]: `.createdBy(rs.getString("createdBy"))`?

**Review by dolbeew — APPROVED** [review]
I'll defer to @marikomedlock on the api code but the ui looks good. Deployed and tested locally.

File: service/src/main/java/bio/terra/tanagra/db/StudyDao.java
Comment (marikomedlock) [inline service/src/main/java/bio/terra/tanagra/db/StudyDao.java:73]: You're not setting the `created` field here because its set to default to `now()` in the liquibase file right? If so, a comment here explaining that would be helpful. ditto for the other dao classes

**Review by marikomedlock — APPROVED** [review]
RE `updateCohortHelper`: Good catch, thanks. Would you mind updating that field here too? I'm also happy to put in the fix in a separate PR, let me know.

LGTM once the 2 field names are fixed.

File: package-lock.json
Comment (melissachang) [reply package-lock.json]: Hmm, I might have accidentally run `npm` in top-level directory. Deleted this file.

File: service/src/main/java/bio/terra/tanagra/db/ReviewDao.java
Comment (melissachang) [reply service/src/main/java/bio/terra/tanagra/db/ReviewDao.java]: Done, good catch. Review was the one artifact I didn't manually test, because I wasn't sure how to create a review in swagger.

File: service/src/main/java/bio/terra/tanagra/db/ReviewDao.java
Comment (melissachang) [reply service/src/main/java/bio/terra/tanagra/db/ReviewDao.java]: Done

File: service/src/main/java/bio/terra/tanagra/db/StudyDao.java
Comment (melissachang) [reply service/src/main/java/bio/terra/tanagra/db/StudyDao.java:74]: Done. I agree comment is better, thanks.

**Comment by melissachang** [comment]
> RE `updateCohortHelper`: Good catch, thanks. Would you mind updating that field here too? I'm also happy to put in the fix in a separate PR, let me know.

I thought I responded to this, looks like the comment got lost or something.

The fix to updateCohortHelper looked non-trivial, so I wanted to talk to you first. I can fix after the holidays, or you can fix before then if you want. :) Concept set had the same problem. I didn't test review or study.

**Pull request merged**
